unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/BuilderInWorldGodMode.cs

```diff
@@ -137,32 +155,89 @@ public override void Init(GameObject goToEdit, GameObject undoGo, GameObject sna
         HUDController.i.buildModeHud.OnScaleSelectedAction += ScaleMode;
     }
 
+    private void MouseDrag(int buttonId, Vector3 mousePosition, float axisX, float axisY)
```

Comment:
If this is some kind of event, it shouldn't be called `OnMouseDrag`? Same with `MouseUp`.